onnxscript/function_libs/torch_aten/ops/nn.py

```diff
@@ -198,10 +198,15 @@ def aten_binary_cross_entropy_backward(
     raise NotImplementedError()
 
 
-def aten_celu(self, alpha: float = 1.0):
+@torch_op("aten::celu")
+def aten_celu(self: TFloat, alpha: float = 1.0, dtype: int = -1) -> TFloat:
```

Comment:
```suggestion
def aten_celu(self: RealType, alpha: float = 1.0, dtype: int = -1) -> FLOAT:
```

since casting is done in the function, we can support any inputs. 

Is dtype part of the aten signature? Could you point me to the doc?

Reply:
> since casting is done in the function, we can support any inputs.
> 
> Is dtype part of the aten signature? Could you point me to the doc?

No, dtype is not a part of the aten signature and it was added just for knowing the input data type so we can cast it back after calling celu which only supports float32.

If we can ask the caller to make sure the input data type, then we don't need to add dtype here and avoid those casting work.